R/model.R

```diff
@@ -237,6 +239,11 @@ compile_method <- function(quiet = TRUE,
              && file.mtime(exe) < file.mtime(self$stan_file())) {
     force_recompile <- TRUE
   }
+  # temporary deprecation warnings
+  if (isTRUE(threads)) {
+    warning("'threads' is deprecated. Please use 'cpp_options = list(stan_threads = TRUE)' instead.")
+    stanc_options[["stan_threads"]] <- TRUE
```

Comment:
should this go into `cpp_options` instead of `stanc_options`?

Reply:
Yes, thanks! Fixed.